### PR TITLE
update update Mapbender

### PR DESCRIPTION
**Pull request by astroidex**
* delete not necessary autdated information

### CODE REVIEW

File: de/installation/update.rst [hunk 1]
```diff
@@ -25,15 +25,15 @@ Im Folgenden sind die einzelnen Schritte als Befehle aufgeführt.
 .. code-block:: bash
 
  # Laden Sie die neue Version herunter
- wget -O http://mapbender.org/builds/mapbender3-3.0.4.0.tar.gz /tmp/build_mapbender/
- tar xfz /tmp/build_mapbender/mapbender3-3.0.4.0.tar.gz
+ wget -O http://mapbender.org/builds/mapbender-starter-current.tar.gz  /tmp/build_mapbender/
```
Comment (rschwammborn) [inline de/installation/update.rst:28]: double space after package link - seems to be a copy/paste error?

File: de/installation/update.rst [hunk 2]
```diff
@@ -54,9 +54,7 @@ Im Folgenden sind die einzelnen Schritte als Befehle aufgeführt.
  app/console doctrine:schema:update --dump-sql
  app/console doctrine:schema:update --force
 
- # Importieren Sie die Anwendungen aus der mapbender.yml Datei, um sich den neusten Stand der Entwicklungen anzuschauen
- app/console doctrine:fixtures:load --fixtures=./mapbender/src/Mapbender/CoreBundle/DataFixtures/ORM/Application/ --append
-
+# Aufbau der symbolischen Links
```
Comment (rschwammborn) [inline de/installation/update.rst:57]: Another minor space error that interrupts the formatting

File: de/installation/update.rst [hunk 3]
```diff
@@ -95,8 +93,7 @@ Aktualisierungsbeispiel für Windows
  #     - ggf. müssen Sie die benötigte Erweiterung auf der Kommandozeile im Aufruf übergeben z.B. 
  #            php -d extension=C:\ms4w\Apache\php\ext\php_pdo_pgsql.dll app/console doctrine:schema:update --dump-sql
   
- # Importieren Sie die Anwendungen aus der mapbender.yml Datei, um sich den neusten Stand der Entwicklungen anzuschauen
- php.exe app/console doctrine:fixtures:load --fixtures=./mapbender/src/Mapbender/CoreBundle/DataFixtures/ORM/Application/ --append
+# Ausspielen in den web-Bereich
```
Comment (rschwammborn) [inline de/installation/update.rst:96]: See above

**Review by rschwammborn — COMMENTED** [review]
Hi, please check the German version of the file, as it contains several space mistakes. English file is fine.

**Comment by astroidex** [comment]
Hi Robin. Should be finde now

**Review by rschwammborn — APPROVED** [review]

**Pull request merged**